hack/build.sh

```diff
@@ -37,19 +39,19 @@ run() {
   export GO111MODULE=on
 
   # Jump into project directory
-  pushd $(basedir) >/dev/null 2>&1
+  pushd "$(basedir)" >/dev/null 2>&1
 
   # Print help if requested
-  if $(has_flag --help -h); then
+  if eval "$(has_flag --help -h)"; then
```

Comment:
@rhuss are you OK with the proposed to solution to fix shellcheck's warning?

Reply:
see my comment above, I would prefer to adapt `$has_flag` to return a 0 or 1 (instead of `true` or `false` that then is interpreted as a UNIX command)

Reply:
@rhuss sure, thanks!

Reply:
@xiangpingjiang sorry for misleading you with my previous comment. If you don't mind to rework it again a bit within Roland's direction.